### ISSUE TITLE
Bug: no-space-in-emphasis does not stop at table cell markers

### ISSUE THREAD

**Issue by stefreak** [igorshubovych/markdownlint-cli#162]
We have a markdown table like this:

Name            | API Name     | Memory | vCPUs | Storage* |
----------------|--------------|--------|-------|----------|
M1 CPU Tiny     |  m1c.tiny    |  2GB   |   1   |   50GB   |
M1 CPU Small    |  m1c.small   |  4GB   |   2   |   50GB   |
M1 CPU Medium   |  m1c.medium  | 8GB    |   4   |   50GB   |
M1 CPU Large    |  m1c.large   | 16GB   |   8   |   50GB   |
M1 CPU XLarge   |  m1c.xlarge  | 32GB   |   16   |   50GB   |
(M1 CPU XXLarge)** |  (m1c.xxlarge)** | 64GB   |   32   |   50GB   |

And markdown lint has this complaint:

```
pages/04.Reference/03.compute/docs.de.md:46:17 MD037/no-space-in-emphasis Spaces inside emphasis markers [Context: "** |  (m1c.xxlarge)**"]
pages/04.Reference/03.compute/docs.en.md:47:17 MD037/no-space-in-emphasis Spaces inside emphasis markers [Context: "** |  (m1c.xxlarge)**"]
```

Expected behaviour: Markdownlint does not complain, because it knows that the stars are not an emphasis, but just plain stars, as it detects the table

**Comment by stefreak** [comment]
We discovered this issue when our renovate wanted to update from v0.20.0 to v0.26.0 and our lint started to fail: https://github.com/syseleven/grav-docs-syseleven-stack/pull/95

**Comment by DavidAnson** [comment]
You should be able to work around this by escaping the asterisk characters like `\*`.

**Comment by stefreak** [comment]
@DavidAnson thanks for the tip!

**Comment by DavidAnson** [comment]
Should be addressed by https://github.com/DavidAnson/markdownlint/commit/d6cd840e7d01658f68b40e217338de23bf6afa6e in the next release of the library.

**Comment by stefreak** [comment]
@DavidAnson nice, thank you. With v0.26.0 now our CI is green again. Nice! :)
